funnel/views/session.py

```diff
@@ -115,26 +116,36 @@ class SessionView(SessionViewMixin, UrlForView, ModelView):
     @render_with('schedule.html.jinja2', json=True)
     def view(self):
         project_save_form = SavedProjectForm()
+        scheduled_sessions_list = session_list_data(
+            self.obj.project.scheduled_sessions, with_modal_url='view_popup'
+        )
         return {
             'project': self.obj.project,
-            'active_session': session_data(self.obj, with_modal_url='view_popup'),
-            'from_date': date_js(self.obj.project.schedule_start_at),
-            'to_date': date_js(self.obj.project.schedule_end_at),
-            'sessions': session_list_data(
-                self.obj.project.scheduled_sessions, with_modal_url='view_popup'
+            'from_date': (
+                localize_timezone(
+                    self.obj.project.schedule_start_at, tz=self.obj.project.timezone
+                ).isoformat()
+                if self.obj.project.schedule_start_at
+                else None
             ),
-            # FIXME: This timezone by UTC offset is not accounting for DST. Look up where it's being used and fix it
-            'timezone': (
-                utcnow()
-                .astimezone(self.obj.project.timezone)
-                .utcoffset()
-                .total_seconds()
+            'to_date': (
+                localize_timezone(
+                    self.obj.project.schedule_end_at, tz=self.obj.project.timezone
+                ).isoformat()
+                if self.obj.project.schedule_start_at
+                else None
             ),
+            'active_session': session_data(self.obj, with_modal_url='view_popup'),
+            'sessions': scheduled_sessions_list,
+            'timezone': self.obj.project.timezone.utcoffset(datetime.now()).total_seconds(),
```

Comment:
Change to `timezone_offset`. Also, use `schedule_start_at` instead of `now`. It's not a fully accurate fix, but it'll work for any date that's not a DST change date.